.github/workflows/push-chart.yml

```diff
@@ -0,0 +1,23 @@
+name: Auto trigger mch-repo
+
+on:
+  push:
+    branches:
+    - release-*
```

Comment:
Don't we want this on release-2.3 and higher?

Reply:
Yes, it will only matter for the current release branch. But if the event gets fired from previous releases it's not going to matter since the chart update script we use in hub-repo is essentially idempotent.